src/components/MediaPlayer/VideoJS/videojs-theme.scss

```diff
@@ -52,135 +62,61 @@
   padding-left: 0;
 }
 
-/* Play action controls */
-.vjs-theme-ramp .vjs-control-bar .vjs-play-control {
-  float: inline-start;
-}
-
-/* Rest of the controls */
-.vjs-theme-ramp .vjs-mute-control,
-.vjs-theme-ramp .vjs-volume-panel {
-  margin-left: auto;
-}
-.vjs-theme-ramp .vjs-picture-in-picture-control,
-.vjs-theme-ramp .vjs-fullscreen-control,
-.vjs-theme-ramp .vjs-track-scrubber,
-.vjs-theme-ramp .vjs-playback-rate.vjs-control,
-.vjs-theme-ramp .vjs-quality-selector,
-.vjs-theme-ramp .vjs-file-download,
-.vjs-theme-ramp .vjs-subs-caps-button {
-  margin-left: 0;
+/* Volume controls */
+.vjs-theme-ramp .vjs-volume-panel .vjs-volume-control.vjs-volume-vertical {
+  margin-top: -1.5em;
 }
 
-/* Reduce height of full-screen toggle for easy scrubbing on progress */
-.vjs-theme-ramp .vjs-control-bar .vjs-fullscreen-control,
-.vjs-theme-ramp .vjs-control-bar .vjs-track-scrubber {
-  margin-top: 0.5em;
-  height: 70%;
-}
+// Make the horizontal volume panel always visible for audio
+.vjs-theme-ramp .vjs-volume-panel.vjs-volume-panel-horizontal {
+  transition: none !important;
+  width: 8em !important;
 
-/* Mute control */
-.vjs-theme-ramp .vjs-mute-control {
-  display: none; // Hide mute control by default
+  // Align volume slider thumb
+  .vjs-volume-horizontal span.vjs-icon-placeholder {
+    margin-top: 0.1em;
+  }
 }
 
-.vjs-theme-ramp.vjs-touch-enabled .vjs-mute-control {
-  margin-left: auto;
-  display: block !important; // Show mute control only on mobile devices
+.vjs-theme-ramp.vjs-audio-only-mode .vjs-volume-panel .vjs-volume-control {
+  opacity: 1 !important;
+  width: 4em !important;
 }
 
 /* Popup-menu button controls */
-.vjs-theme-ramp .vjs-menu-button>.vjs-icon-placeholder {
-  margin-top: 0.5em;
-}
-
-.vjs-theme-ramp .vjs-menu-button-popup .vjs-menu {
-  margin-bottom: 0.75em;
-}
-
-.vjs-theme-ramp div.vjs-menu-button {
-  height: 60%;
-  margin-top: 0.5em;
-}
-
-/* Volume stuff */
-.vjs-theme-ramp .vjs-volume-panel:hover .vjs-volume-control.vjs-volume-horizontal {
-  height: 100%;
+.vjs-theme-ramp .vjs-menu li.vjs-menu-title {
+  text-transform: capitalize;
 }
```

Comment:
I'm thinking that we need to keep hover display of the volume popup since click toggles mute or am I not understanding this correctly?

Reply:
Hover display still works for the vertical volume panel. The removed CSS here was for the triangle volume bar in the previous implementation.